src/projects/project-pull-request-list-item.html

```diff
@@ -86,13 +86,18 @@
         issue: Object,
         showStatus: Boolean,
         selected: Boolean,
+        statuses: Array,
         _cistatus: {
           type: String,
           computed: '_getCIStatus(statuses)'
         },
         _updatedAtString: {
           type: String,
           computed: 'toDateString(pr.updated_at)'
+        },
+        _latestStatusUrl: {
+          type: String,
+          computed: '_getLatestStatusUrl(statuses)'
```

Comment:
I dont really get how to do this, because of the " then modify the binding in the class above".